src/index.js

```diff
@@ -12,6 +13,7 @@ const formatsLogger = app.get("env") === "development" ? "dev" : "short";
 app.use(logger(formatsLogger));
 app.use(cors());
 app.use(express.json());
+app.use(ourLogger);
```

Comment:
Did you test if it works. I'm not sure that it is. It incorrectly invoked here

Reply:
I corrected, sorry.